bin/compile

```diff
@@ -77,16 +77,9 @@ start "Installing Dependencies"
 shards install --production
 finished
 
-if [ -f app.cr ]; then
-    start "Compiling app.cr (using old behaviour, app.cr exists - ignoring shard.yml)"
-    crystal build app.cr --release $NO_DEBUG_IF_AVAILABLE
-    finished
-else
-    eval $(parse_yaml shard.yml "shard_")
-    start "Compiling src/${shard_name}.cr (auto-detected from shard.yml)"
-    crystal build src/${shard_name}.cr --release -o app
-    finished
-fi
+start "Running shards builds --release"
+shards build --release
```

Comment:
Oh, does this need the `--production` flag? `shards build --release --production`. I think that is so it doesnt' build ameba targets and stuff like that...